client/src/modules/entities/entities.html

```diff
@@ -0,0 +1,45 @@
+<div class="flex-header">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static" translate> TREE.ADMIN </li>
+      <li class="title" translate> ENTITY.MANAGEMENT </li>
+    </ol>
+
+    <div class="toolbar">
+      <div class="toolbar-item">
+        <button class="btn btn-default text-capitalize" id="create" ng-click="EntityCtrl.createEntity()" data-method="create">
+          <span class="fa fa-plus"></span> <span translate>ENTITY.ADD_ENTITY</span>
+        </button>
+      </div>
+
+      <div class="toolbar-item">
+        <button
```

Comment:
Can you use the `bhFilterTogger` here?

Reply:
This would be a good use for our `bhFilterToggle`.  You can use it like this:
```html
<bh-filter-toggle
  on-toggle="EntityCtrl.toggleFilter()">
</bh-filter-toggle>
```